src/core/storage/compression/Decompressor.h

```diff
@@ -215,11 +244,10 @@ class DecompressorImpl {
 				fread(&m_current_byte, 1, 1, m_compressed);
 				m_current_bit_count = CHAR_BIT;
 			}
-			if (m_current_byte & Check) {
+			if (m_current_byte & Check)
 				(*node) = (*node)->one;
-			} else {
```

Comment:
Please do not make whitespace changes when committing patches with functional modifications.